sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2WriteSupportCheck.scala

```diff
@@ -18,8 +18,8 @@
 package org.apache.spark.sql.execution.datasources.v2
 
 import org.apache.spark.sql.AnalysisException
-import org.apache.spark.sql.catalyst.expressions.Literal
-import org.apache.spark.sql.catalyst.plans.logical.{AppendData, LogicalPlan, OverwriteByExpression, OverwritePartitionsDynamic}
+import org.apache.spark.sql.catalyst.expressions.{Literal, SubqueryExpression}
+import org.apache.spark.sql.catalyst.plans.logical._
```

Comment:
Minor: do not replace imports with wildcard imports. It causes git commit conflicts and can cause namespace conflicts later.

Reply:
Thank you @rdblue . Have fixed it.